docker/Dockerfile

```diff
@@ -32,4 +32,4 @@ RUN git clone https://github.com/apache/nutch.git nutch_source && \
      rm -rf /root/.ivy2/
 
 # Convenience symlink to Nutch runtime local
```

Comment:
The comment should reflect that the symlink now points to the script bin/nutch.

Just two ideas:
- defining the environment variable NUTCH_HOME could be useful:
  ```
  env NUTCH_HOME=`/root/nutch_source/runtime/local`
  ```
- why not put symlinks to bin/nutch (and bin/crawl) into /usr/local/bin to make them available from everywhere?
  ```
  ln -sf $NUTCH_HOME/bin/nutch /usr/local/bin/
  ln -sf $NUTCH_HOME/bin/crawl /usr/local/bin/
 ```

Reply:
Excellent points @sebastian-nagel I'll implement them. Thanks